Jenkinsfile

```diff
@@ -72,6 +72,17 @@ node {
           rsync -arv -e \"ssh -2\" ./chrome/*.json sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}/chrome
         """
       }
+
+      // set akamai fast purge env
+      openShiftUtils.withJnlpNode(image: "quay.io/redhatqe/jenkins-slave-python36-akamai:v3.11") {
+        sh "wget https://raw.githubusercontent.com/RedHatInsights/cloud-services-config/ci-beta/akamai/cache_buster/bust_cache.py"
```

Comment:
Should we make this `env` aware? E.g. pulling from prod-beta/prod-stable instead of always ci-beta.

Reply:
We could. But I don't know if there will be any differences really.